code/game/objects/items/stacks/sheets/alloy.dm

```diff
@@ -0,0 +1,99 @@
+/* Alloys
+ * Alloys have different properties depending on their composition
+ * The "strength" of these properties depends on the metal/glass:mineral ratio
+ * A 50/50 ratio gives full strength properties
+ *
+ * Mineral effects:
+ * 		Diamond - Physically strengthens the alloy
+ *		Uranium - The alloy emits radiation
+ *		Phoron - Considerably increased heat resistance
+ *		Gold
+ *		Silver
+ *		Iron - Weaker version of diamond
+ *		Platinum
+ *		Tritium
+ *		Osmium
+*/
+
+var/list/prefix = list("diamond" = "ada", "uranium" = "ura", "solid phoron" = "phoro", "gold" = "dives", "silver" = "argent", "iron" = "ferro", "platinum" = "cata", "tritium" = "trit", "osmium" = "osi")
+var/list/postfix = list("metal" = "metallic", "glass" = "glaseous")
+
+/*
+ * Alloy sheet
+*/
+/obj/item/stack/sheet/alloy
+	name = "alloy"
+	desc = "Synthesized alloys with special properties."
+	singular_name = "alloy sheet"
+	icon = 'icons/obj/items/materials.dmi'
+	icon_state = "polysteel-1"
+	item_state = "polysteel"
+	var/list/materials = list()
+
+/obj/item/stack/sheet/alloy/New(var/list/comp)
+	..()
+	if(!comp)
+		usr << "\red This should never appear. There HAS to be a composition list!"
```

Comment:
Make sure not to use text macros like \red for colors, instead use <span class='warning'> at the beginning of the message and </span> at the end